js/app.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+
+document.getElementById('requestResourceButton').addEventListener('click', requestResource);
+
+function requestResource(){
+  const userInput = document.getElementById('resourceId').value;
+  const resourceType = document.getElementById('resourceType');
+  const resourceReq = new XMLHttpRequest();
+
+  if (resourceType.value === 'people'){
+    resourceReq.addEventListener('load', loadPeopleResource);
+    resourceReq.open('GET', `https://swapi.co/api/people/${userInput}`);
+    resourceReq.send();
+    return;
+  }
+  if (resourceType.value === 'planets'){
+    resourceReq.addEventListener('load', loadPlanetResource);
+    resourceReq.open('GET', `https://swapi.co/api/planets/${userInput}`);
+    resourceReq.send();
+    return;
+  }
+  if (resourceType.value === 'starships'){
+    resourceReq.addEventListener('load', loadStarshipResource);
+    resourceReq.open('GET', `https://swapi.co/api/starships/${userInput}`);
+    resourceReq.send();
+    return;
+  }
+}
+
+function loadPeopleResource(){
+  const resourceObj = JSON.parse(this.responseText);
+  const contentContainerDiv = document.getElementById('contentContainer');
+  contentContainerDiv.innerHTML = "";
+
+  if (!resourceObj.name){
+    const errorElem = document.createElement('p');
+    errorElem.innerHTML = 'No person found, try another value';
+    contentContainerDiv.appendChild(errorElem);
+    return;
+  }
+
+  const resourceNameElem = document.createElement('h2');
```

Comment:
Generally we want to keep the code for the creation of the element, the setting of the innerHTML, and the appending of the element to something already on the DOM together. 
For example: 
```
const resourceNameElem = document.createElement('h2');
resourceNameElem.innerHTML = resourceObj.name;
contentContainerDiv.appendChild(resourceNameElem);
```

Rather than split out by the methods (createElement, innerHTML, appendChild).